lsp-rust.el

```diff
@@ -335,6 +335,11 @@ PARAMS progress report notification data."
   :type 'boolean
   :package-version '(lsp-mode . "6.2"))
 
+(defcustom lsp-rust-analyzer-max-inlay-hint-length 2
+  ""
```

Comment:
Can you please add the docstring?

BTW, what is this for? I don't see it's called.

Reply:
This PR is WIP, sorry for not mentioning.